src/middleware.mjs

```diff
@@ -94,7 +94,7 @@ export const processRequest = (
               operationsPath.set(path, map.get(fieldName).promise)
           }
 
-          resolve(operations)
+          resolve({ operations, map })
```

Comment:
This is a breaking change to the API. A fair few people [such as Apollo](https://github.com/apollographql/apollo-server/blob/4405405/packages/apollo-server-express/src/ApolloServer.ts#L12) use `processRequest` directly.

Reply:
That's a good point; I will change how this works.